lib/capistrano/slack_notify.rb

```diff
@@ -117,6 +118,11 @@ def self.extended(configuration)
             set(:start_time, Time.now)
           end
 
+          desc "Notify Slack that the rollback has completed."
+          task :rolled_back do
+            post_to_channel(:yellow, "#{deployer} has rolled back #{deploy_target}")
```

Comment:
@parkr That makes sense to me. My previous thinking was that something out of the ordinary has occurred and passing green, one would hope for development environments, ordinary :)